draft-ion.adoc

```diff
@@ -52,50 +58,59 @@ However, JSON's simple grammar can support virtually any data model - additional
 context is required for machine clients and servers to interpret data and
 metadata in important ways.
 
-Ion exists to codify specific name/value pairs within a JSON object to better
-facilitate JSON-based data and metadata exchange between machine clients and
-servers. As such, Ion does not define a new data grammar - it is a
-Media Subtype of `application/json`.
+Ion codifies specific name/value pairs within JSON objects to better
+facilitate JSON-based data and metadata exchange between machines.  And while machine interchange is the primary goal, Ion's second most important design goal is to enable intuitive and context-relevant structure so Ion content is very easy to understand for humans, even if they do not know what Ion is.  As such, Ion does not define a new data grammar - it is a Media Subtype of `application/json`.
 
 === Terminology
 
 These terms are defined by this specification:
 
+.Member
+A JSON name/value pair as defined in https://tools.ietf.org/html/rfc7159#section-4[RFC 7159].
+
 .Metadata
-Data about a resource.
+A set of data that provides information about other data.
 
 .Meta Object
-A JSON object that contains resource metadata.
+A JSON object with one or more members that all represent Metadata.
+
+.Root Object
+The single JSON object at the root of an Ion content structure.
```

Comment:
Hm. Is the point of the _Root Object_ that you must not do `{[]}`? That is, only object variants of JSON are ION? I had been wondering about this. If so, wonder if it might be useful to expound a little as it seems a little strange or like no-duh otherwise or missing the value of this. 

I think it is an important point and actually how all JSON responses should be from APIs so you can put in meta data, but that is another discussion.